src/store.ts

```diff
@@ -418,6 +445,14 @@ export default function Store(
             }
         }
     }
+
+    this.getDeviceId = () => this.deviceId;
+    this.setDeviceId = (deviceId: string) => {
+        this.deviceId = deviceId;
+        this.persistenceData.gs.das = deviceId;
+
+        mpInstance._Persistence.savePersistence(this.persistenceData);
```

Comment:
I originally thought that since nothing else is saving to the store persistence, that if the user ever set the deviceid, that only the deviceId would be saved the the store, and that everything else would be null.  Then I saw that this was merged into a feature branch, so I think we just need to be 100% sure before merging feature branch into development that every item in Store.persistenceData is filled out and matches persistence.